### PR TITLE
Added line scrubber component to mapchip

### PR DESCRIPTION
**Pull request by goyal1092**
## Description

Line scrubber component to mapchip

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-407

## How is it tested automatically?
Added a gui test

## How should a reviewer test it locally
enable linear scrubber from admin panel for a profile indicator
select that PI in FE 
Linear scrubber should be visible on mapchip when any subindicator from that PI is selected

## Screenshots
Screenshot 2022-08-04 at 11 24 18 AM

### Added
Added a new react component
Added a new trigger event

### Updated
failing unit test to not use onSubindicatorChangr

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/elements/mapchip/components/linear_scrubber/rendrer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import Scrubber from './scrubber';
+import {Component} from "../../../../utils";
+import {Tooltip} from "../../../../ui_components/tooltip";
+
+export class LinearScrubberRenderer extends Component {
```
Comment (emre2038) [inline src/js/elements/mapchip/components/linear_scrubber/rendrer.js:7]: fix the typo in the file name please
